src/main/kotlin/org/octopusden/octopus/oc/template/plugins/gradle/service/OcTemplateService.kt

```
@@ -254,15 +262,89 @@ abstract class OcTemplateService @Inject constructor(
+                streamingProcesses[podName] = process
+                process.inputStream.use { input ->
+                    logFile.outputStream().use { output ->
+                        input.copyTo(output)
+                    }
+                }
+                val exitCode = process.waitFor()
+                if (exitCode == 0) return
+            } catch (_: InterruptedException) {
+                Thread.currentThread().interrupt()
+                return
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)